miio/tests/test_device.py

```diff
@@ -182,3 +182,17 @@ def test_supports_miot(mocker):
 
     send.side_effect = None
     assert d.supports_miot() is True
+
+
+@pytest.mark.parametrize("getter_name", ["actions", "settings", "sensors"])
+def test_cached_descriptors(getter_name, mocker):
+    d = Device("127.0.0.1", "68ffffffffffffffffffffffffffffff")
+    getter = getattr(d, getter_name)
+    initialize_descriptors = mocker.spy(d, "_initialize_descriptors")
+    mocker.patch("miio.Device.status")
+    mocker.patch("miio.Device._sensor_descriptors_from_status", return_value={})
+    mocker.patch("miio.Device._setting_descriptors_from_status", return_value={})
+    mocker.patch("miio.Device._action_descriptors", return_value={})
+    for _i in range(5):
```

Comment:
Just for the future, the proper way to do this is to use plain `_` here. No need to fix that now though.